src/chat/convert-to-openrouter-chat-messages.ts

```diff
@@ -186,6 +186,10 @@ export function convertToOpenRouterChatMessages(
               reasoningDetails.push({
                 type: ReasoningDetailType.Text,
                 text: part.text,
+                id: null,
+                format: ReasoningFormat.AnthropicClaudeV1,
```

Comment:
This one is still a bit dangerous, because we need to also send reasoning back to other format as well